robot/framesystem/framesystem.go

```diff
@@ -77,24 +90,41 @@ func RobotFsCurrentInputs(
 // New returns a new frame system service for the given robot.
 func New(ctx context.Context, r robot.Robot, logger golog.Logger) Service {
 	return &frameSystemService{
+		Named:  InternalServiceName.AsNamed(),
 		r:      r,
 		logger: logger,
 	}
 }
 
+var internalFrameSystemServiceName = resource.NewName(
+	internal.ResourceNamespaceRDKInternal,
+	resource.ResourceTypeService,
+	"framesystem",
+	"builtin",
```

Comment:
I noticed theres a couple instances in this PR using "builtin" as a hardcoded string literal.  `resource.DefaultServiceName` exists and can be used here

Reply:
Yep